Tasks/CopyFilesV2/Tests/L0cleansIfSpecified.ts

```diff
@@ -41,9 +41,9 @@ runner.registerMockExport('stats', (itemPath: string) => {
     }
 });
 let origReaddirSync = fs.readdirSync;
-fs.readdirSync = (p: string | Buffer) => {
+fs.readdirSync = (p) => {
     console.log('HERE path ' + p);
-    let result: string[];
+    let result;
```

Comment:
Suppose, you can delete that variable at all, and use `return` in `if-else` construction